src/metrics.jl

```diff
@@ -208,46 +208,46 @@ kl_divergence(a::AbstractArray, b::AbstractArray) = evaluate(KLDivergence(), a,
 
 # RenyiDivergence
 function eval_start{T<:AbstractFloat}(::RenyiDivergence, a::AbstractArray{T}, b::AbstractArray{T})
```

Comment:
Honestly, these argument names would have to be changed throughout the code for a lot of the distance metrics - I can do that, but it'll look like I wrote the whole package... I wasn't sure it was worth it, but I'm happy to do it if you like?